test-apps/pods-embroider-app/package.json

```diff
@@ -45,17 +45,14 @@
     "ember-cli-dependency-checker": "^3.3.2",
     "ember-cli-htmlbars": "^6.3.0",
     "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-data": "~5.3.0",
```

Comment:
these dependencies aren't needed, and made debugging more strenuous when having breakpoints all over the place